docs/js/.nojekyll

```diff
@@ -0,0 +1 @@
+TypeDoc added this file to prevent GitHub Pages from using Jekyll. You can turn off this behavior by setting the `githubPages` option to false.
```

Comment:
We should probably do what is suggested here and set the `githubPages` option to false as we do not intend to use GH Pages in any foreseeable future.